exercises/michael_turner/ex01_basics/src/counter.h

```diff
@@ -0,0 +1,33 @@
+#ifndef COUNTER_H_
+#define COUNTER_H_
+
+//Includes
+#include <iosfwd>
+#include <map>
+#include <string>
+
+const std::string::size_type  MIN_LENGTH = 5;
+const std::string PUNC = ".,?'\"!():;";
+const std::string HYPHEN = "-";
+const int OUT_WIDTH = 20;
+
+typedef std::map<std::string, int> wordMap;
+typedef std::multimap<int, std::string> wordRank;
+typedef std::string::size_type string_size;
+
```

Comment:
Also, the naming convention for typedefs is PascalCase, e.g. `WordMap` instead of `wordMap`